DOCS/man/mpv.rst

```diff
@@ -1501,6 +1501,30 @@ For Windows-specifics, see `FILES ON WINDOWS`_ section.
     in default configuration will use ``/usr/local/etc/mpv/`` as config
     directory, while most Linux distributions will set it to ``/etc/mpv/``).
 
+``~/.config/mpv``
+    The standard configuration directory. This can be overridden by environment
+    variables:
+
+    If ``$XDG_CONFIG_HOME`` is set, then the derived configuration directory
+    will be ``$XDG_CONFIG_HOME/mpv``.
+
+    If ``$HOME`` is set, then the derived configuration directory will be
+    ``$HOME/.config/mpv``.
+
+    If ``$MPV_HOME`` is set, then the derived configuration directory will be
+    ``$MPV_HOME``.
+
+    If the directory, nor the original configuration directory (see below) do
+    not exist, mpv tries to create and use it.
+
+``~/.mpv/``
+    The original (pre 0.5.0) configuration directory. It will continue to be
+    read if present.
+
+    If both this directory and the standard configuration directory are
+    present, configuration will be read from both with the standard
+    configuration directory content taking precedence.
```

Comment:
Since 22a27c6720244dd24c9b704c8a8225ca1665272f there's a warning if this is the case, might be worth noting here